### PR TITLE
Add resize for segmentation

### PR DESCRIPTION
**Pull request by zhangxiaoli73**
## What changes were proposed in this pull request?

Add resize for segmentation

## How was this patch tested?

unit tests

## Related links or issues (optional)
fixed https://github.com/intel-analytics/BigDL/issues/XXX

### CODE REVIEW

File: spark/dl/src/main/scala/com/intel/analytics/bigdl/dataset/segmentation/Resize.scala [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+package com.intel.analytics.bigdl.dataset.segmentation
+
+import com.intel.analytics.bigdl.transform.vision.image.{FeatureTransformer, ImageFeature}
+import com.intel.analytics.bigdl.transform.vision.image.label.roi.RoiLabel
+import com.intel.analytics.bigdl.transform.vision.image.util.BboxUtil
+import org.opencv.core.Size
+import org.opencv.imgproc.Imgproc
+
+class Resize(minSize: Int, maxSize: Int = -1, resizeROI: Boolean = false)
```
Comment (wzhongyuan) [inline spark/dl/src/main/scala/com/intel/analytics/bigdl/dataset/segmentation/Resize.scala]: Can we name it a different one and put it under the same package as other transformer ?

Comment (zhangxiaoli73) [reply spark/dl/src/main/scala/com/intel/analytics/bigdl/dataset/segmentation/Resize.scala]: ok, changed

**Comment by wzhongyuan** [comment]
LGTM

**Comment by zhangxiaoli73** [comment]
jenkins pass: http://mlt-bddl01.sh.intel.com:8080/job/BigDL-PRVN-lite/971/

**Pull request merged**